pkg/sqlfs/sql_writer_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlfs
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSQLWriter(t *testing.T) {
+	testDriver = getEnv("SQLFLOW_TEST_DB", "mysql")
+	if testDriver == "hive" {
+		return // For Hive, please use HiveWriter instead.
+	}
+	t.Logf("Confirm executed with %s", testDriver)
+
+	a := assert.New(t)
+
+	fn := fmt.Sprintf("%s.unitest%d", testDatabaseName, rand.Int())
+	w, e := newSQLWriter(testDB, testDriver, fn)
+	a.NoError(e)
+	a.NotNil(w)
+	defer w.Close()
+
+	has, e1 := hasTable(testDB, fn)
+	a.NoError(e1)
+	a.True(has)
+
+	a.NoError(dropTable(testDB, fn))
+}
+
+func TestSQLWriterWriteAndRead(t *testing.T) {
+	testDriver = getEnv("SQLFLOW_TEST_DB", "mysql")
+	if testDriver == "hive" {
+		return // For Hive, please use HiveWriter instead.
```

Comment:
Can use `t.Skip()` instead of `return` so that other developers can know to skip this test case clearly.

Reply:
Thanks for the remind! Will update in successive PRs.